{{cookiecutter.bot_name}}/app/bot/commands/common.py

```diff
@@ -49,3 +53,11 @@ async def show_help(message: Message, bot: Bot) -> None:
 async def git_commit_sha(message: Message, bot: Bot) -> None:
     """Show git commit SHA."""
     await bot.answer_message(environ.get("GIT_COMMIT_SHA", "<undefined>"), message)
+
+
+@collector.hidden(command="/_history")
+async def history(message: Message, bot: Bot) -> None:
+    """Show history of unhandled messages."""
+    records = await Record.all()
+    text = "\n".join(row[0].record_data for row in records)
```

Comment:
Use `Records.all().scalars()`. This will cast `list[tuple[Record]]` to `list[Record]`.